### ISSUE TITLE
Improve Cairo´s handeling of Administrator Apps

### ISSUE THREAD

**Issue by PeterStrick** [cairoshell/cairoshell#354]
**Is your feature request related to a problem? Please describe.**
Currently, when you often start an Application as Administrator, Cairo will ask you if said Program should always run as an Administrator. The problem is if you don´t want the application to always run as Admin anymore.

**Describe the solution you'd like**
Having a section for that in Cairo somewhere like in the Setting or rightclicking the Application in the Programs menu or Quick Lanch, that has a checkmark that says "Always run as Administrator".

**Describe alternatives you've considered**
Since uninstalling and reinstalling the Application that alwys runs as Admin doesn´t work because it is gonna run as Admin again, the only way to stop this is to reset Cairo.

**Additional context**
/

**Comment by dremin** [comment]
Oops, you can adjust this by editing `%appdata%\CairoAppConfig.xml` but I agree this needs to be a configurable option in the context menu like you suggested. Thanks for bringing this up!

**Comment by PeterStrick** [comment]
> Oops, you can adjust this by editing `%appdata%\CairoAppConfig.xml` but I agree this needs to be a configurable option in the context menu like you suggested. Thanks for bringing this up!

Oh ok. Didn´t knew that. But you should be able to just right click the application and have a setting there like i suggested. It would make life a bit easier.

**Comment by dremin** [comment]
I agree, I am going to re-open this so that I don't forget to add that :)
